docs/install/config-ha.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 50
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+HA functionality is available by default on Knative for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components. These components are configured with two replicas by default.
```

Comment:
I think `kourier` is redundant here? Or we need to list istio as well (both pilot and IGW are HA components).

Reply:
We could list Istio too. Is the component just `istio-gateway`? @vagababov

Reply:
both istio-gateway and istio-pilot are H(P)A components

Reply:
`HA functionality is available by default on Knative for the ` this is also a bit hard to parse:
since it might be percieved as _default on_ which it isn't.